components/GothamistMainHeader.vue

```diff
@@ -1,134 +1,166 @@
 <script setup lang="ts">
-import GothamistSidebarContents from "./GothamistSidebarContents.vue";
-import Navigation from "~~/composables/types/Navigation";
+import GothamistSidebarContents from './GothamistSidebarContents.vue'
+import Navigation from '~~/composables/types/Navigation'
 import VFlexibleLink from '@nypublicradio/nypr-design-system-vue3/v2/src/components/VFlexibleLink.vue'
 
 defineProps<{
-    showLogo: boolean,
-    navigation: Navigation
-    donateUrl: string
+  showLogo: boolean
+  navigation: Navigation
+  donateUrl: string
 }>()
 
-const sideBarVisible = ref(false);
+const sideBarVisible = ref(false)
+
+const menuListClick = (menuTitle) => {
```

Comment:
added a method to fire on menu-list and menu logo clicks. I am using emits to capture these click events. I'll point them out below